_posts/events/2020-06-28-linaro-virtual-connect-2020.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Linaro Virtual Connect 2020 "
+event: linaro-virtual-connect
+event_url: "connect.linaro.org "
```

Comment:
Hi @kristinedill, 
The event_url should contain the protocol. i.e https://connect.linaro.org
BR,
Kyle